src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/UnnecessaryParenthesesCheck.java

```diff
@@ -205,6 +209,16 @@
      */
     public static final String MSG_LAMBDA = "unnecessary.paren.lambda";
 
+    /**
+     * Compiled pattern used to match newline control characters, for replacement.
+     */
+    private static final Pattern NEWLINE = Pattern.compile("\n");
```

Comment:
Should this use `\R`?

Reply:
Yes, that seems like it always a good idea. I will always do this from now on.